<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDbQuickMappingWizard</class>
 <widget class="QWizard" name="CDbQuickMappingWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>swift mapping wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <widget class="BlackGui::COverlayMessagesWizardPage" name="wp1_Intro">
   <property name="title">
    <string>Model</string>
   </property>
   <property name="subTitle">
    <string>First description</string>
   </property>
   <layout class="QVBoxLayout" name="vl_IntroPage">
    <item>
     <widget class="QFrame" name="fr_Introduction">
      <layout class="QHBoxLayout" name="hl_Introduction" stretch="0,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_IntroIcon">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../misc/misc.qrc">:/own/icons/own/swift3D/sw3DGreen-48.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_IntroText">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You are about to write a new mapping for your current model to the swift DB. Hereby you agree with the swift CLA and license terms. For details please consult the swift about and legal page. &lt;/p&gt;&lt;p&gt;Thank you for supporting swift&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vl_Intro">
      <item alignment="Qt::AlignTop">
       <widget class="QGroupBox" name="gb_Model">
        <property name="title">
         <string>Model</string>
        </property>
        <layout class="QVBoxLayout" name="vl_Model">
         <item>
          <widget class="QLineEdit" name="le_ModelString">
           <property name="whatsThis">
            <string>the unique key identifies each aircraft model</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>model`s unique key</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_AircraftIcao">
        <property name="title">
         <string>Aircraft ICAO code</string>
        </property>
        <layout class="QVBoxLayout" name="vl_AircraftIcao">
         <item>
          <widget class="BlackGui::Components::CDbAircraftIcaoSelectorComponent" name="selector_AircraftIcaoCode">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wi_AircraftIcaoHints" native="true">
           <layout class="QHBoxLayout" name="hl_AircraftIcaoHints">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>30</number>
            </property>
            <item>
             <widget class="QLabel" name="lbl_HintAircraftIcaoIcon">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../misc/misc.qrc">:/pastel/icons/pastel/16/infomation.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_HintAircraftIcao">
              <property name="text">
               <string>Hint: You can also search for the aircraft ICAO on the next page.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gb_AirlineIcao">
        <property name="title">
         <string>Airline</string>
        </property>
        <layout class="QVBoxLayout" name="vl_AirlineIcao">
         <item>
          <widget class="QLabel" name="lbl_AirlineOrNoAirline">
           <property name="text">
            <string>Does the aircraft belong to an airline?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wi_AirlineOrNoAirline" native="true">
           <layout class="QVBoxLayout" name="vl_AirlineOrNoAirline">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QRadioButton" name="rb_ColorLivery">
              <property name="text">
               <string>no, this is a (small) aircraft without airline livery. Use a color livery instead.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_AirlineLivery">
              <property name="text">
               <string>yes, it belongs to an airline or Air Force</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="wi_HintAirlineIcao" native="true">
           <layout class="QHBoxLayout" name="hl_HintAirlineIcao">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="lbl_HintIconAirlineIcao">
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../../misc/misc.qrc">:/pastel/icons/pastel/16/infomation.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lbl_HintAirlineIcao">
              <property name="text">
               <string>Hint: Start virtual airlines with &quot;V&quot;, e.g. &quot;VLHA&quot;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="fr_IntroAirlineSelection">
           <layout class="QFormLayout" name="fl_AircraftIcaoIntro">
            <property name="horizontalSpacing">
             <number>12</number>
            </property>
            <item row="2" column="0">
             <widget class="QLabel" name="lbl_AirlineSelector">
              <property name="text">
               <string>Airline ICAO</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="BlackGui::Components::CDbAirlineIcaoSelectorComponent" name="selector_AirlineIcaoCode">
              <property name="maximumSize">
               <size>
                <width>225</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lbl_AirlineName">
              <property name="text">
               <string>Airline name</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="BlackGui::Components::CDbAirlineNameSelectorComponent" name="selector_AirlineName">
              <property name="maximumSize">
               <size>
                <width>225</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="lbl_VirtualAirline">
              <property name="text">
               <string>Virtual airline?</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="cb_VirtualAirline">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="lbl_Military">
              <property name="text">
               <string>Military?</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QCheckBox" name="cb_Military">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="BlackGui::COverlayMessagesWizardPage" name="wp2_Aircraft">
   <property name="title">
    <string>Aircraft</string>
   </property>
   <property name="subTitle">
    <string>Please select the ICAO code which best represents yout model</string>
   </property>
   <layout class="QVBoxLayout" name="vl_AircraftPage">
    <item>
     <widget class="BlackGui::Components::CDbAircraftIcaoComponent" name="comp_AircraftIcao">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="BlackGui::COverlayMessagesWizardPage" name="wp3_LiveryColor">
   <property name="title">
    <string>Livery</string>
   </property>
   <property name="subTitle">
    <string>Livery by color</string>
   </property>
   <layout class="QVBoxLayout" name="vl_LiveryColorPage">
    <item>
     <widget class="QFrame" name="fr_ColorInfo">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="vl_ColorInfo">
       <item>
        <widget class="QLabel" name="lbl_ColorInfo">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your aircraft does not belong to an airline.&lt;br/&gt;Select its fuselage and tail color (or its primary and secondary color).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_ColorSearch">
      <property name="title">
       <string>Select color</string>
      </property>
      <layout class="QVBoxLayout" name="vl_ColorSearch">
       <item>
        <widget class="BlackGui::Components::CDbLiveryColorSearch" name="comp_ColorSearch">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="BlackGui::COverlayMessagesWizardPage" name="wp4_Livery">
   <property name="title">
    <string>Livery</string>
   </property>
   <property name="subTitle">
    <string>Please select the livery best matching your aircraft</string>
   </property>
   <layout class="QVBoxLayout" name="vl_LiveryPage">
    <item>
     <widget class="BlackGui::Components::CDbLiveryComponent" name="comp_Livery">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="BlackGui::COverlayMessagesWizardPage" name="wp5_Distributor">
   <property name="title">
    <string>Distributor / package</string>
   </property>
   <property name="subTitle">
    <string>Select the correct package / distributor. HINT: Use 'ZZZZ' if there is no corresponding simulator-</string>
   </property>
   <layout class="QVBoxLayout" name="vl_DistributorPage">
    <item>
     <widget class="BlackGui::Components::CDbDistributorComponent" name="comp_Distributor">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>250</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="BlackGui::COverlayMessagesWizardPage" name="wp6_Confirmation">
   <property name="title">
    <string>Confirmation</string>
   </property>
   <property name="subTitle">
    <string>Check if the data are correct</string>
   </property>
   <layout class="QVBoxLayout" name="vl_ConfirmationPage">
    <item>
     <widget class="QGroupBox" name="gb_SupportedSimulators">
      <property name="title">
       <string>Supported simulators</string>
      </property>
      <layout class="QHBoxLayout" name="hl_SupportedSimulators">
       <item alignment="Qt::AlignLeft">
        <widget class="BlackGui::Components::CSimulatorSelector" name="selector_Simulator">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="BlackGui::Editors::CAircraftModelForm" name="editor_AircraftModel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="fr_ConfirmationOk">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lbl_Confirmation">
         <property name="text">
          <string>You can now send the data to the database. Press the &quot;Next&quot; button.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="fr_ConfirmationStillErrors">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lbl_StillErrors">
         <property name="text">
          <string>There are still errors. Use the &quot;Back&quot; button and review your selections.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="BlackGui::COverlayMessagesWizardPage" name="wp7_Credentials">
   <property name="title">
    <string>Credentials</string>
   </property>
   <property name="subTitle">
    <string>Please tell us who you are</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="gb_Credentials">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>125</height>
       </size>
      </property>
      <property name="title">
       <string>User data</string>
      </property>
      <layout class="QVBoxLayout" name="vl_Credentials">
       <item>
        <widget class="BlackGui::Components::CDbLoginComponent" name="comp_DbLogin">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="BlackGui::COverlayMessagesWizardPage" name="wp8_LastConfirmation">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="lbl_Info">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;By pressing &amp;quot;NEXT&amp;quot; you will send the changed data!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="BlackGui::COverlayMessagesWizardPage" name="wp9_SendStatus">
   <property name="title">
    <string>Status</string>
   </property>
   <property name="subTitle">
    <string>Overall status confirmation</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gb_SendStatus">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>125</height>
       </size>
      </property>
      <property name="title">
       <string>Send status</string>
      </property>
      <layout class="QVBoxLayout" name="vl_SendStatus">
       <item>
        <widget class="BlackGui::Components::CStatusMessagesDetail" name="comp_Log">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CDbDistributorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbdistributorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAircraftIcaoSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbaircrafticaoselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAirlineIcaoSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbairlineicaoselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbLiveryColorSearch</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbliverycolorsearch.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAircraftIcaoComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbaircrafticaocomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbLiveryComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbliverycomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAirlineNameSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbairlinenameselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CStatusMessagesDetail</class>
   <extends>QFrame</extends>
   <header>blackgui/components/statusmessagesdetail.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CAircraftModelForm</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/aircraftmodelform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbLoginComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dblogincomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::COverlayMessagesWizardPage</class>
   <extends>QWizardPage</extends>
   <header>blackgui/overlaymessagesframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_ModelString</tabstop>
  <tabstop>selector_AircraftIcaoCode</tabstop>
  <tabstop>rb_ColorLivery</tabstop>
  <tabstop>rb_AirlineLivery</tabstop>
  <tabstop>selector_AirlineIcaoCode</tabstop>
  <tabstop>selector_AirlineName</tabstop>
  <tabstop>cb_VirtualAirline</tabstop>
  <tabstop>cb_Military</tabstop>
 </tabstops>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
